### DISCUSSION TITLE
ppgw.ini 能否新增设置选项 剔除 yaml 或者suburl 中的 DIRECT 规则。

### DISCUSSION THREAD

**Discussion by ghost** [kkkgo/PaoPaoGateWay#61] [想法]
在经过DNS分流之后 进入到PPGW中的流量默认已经是国外流量了。
但是有些机场订阅的yaml中有直连规则。
今天就遇到 fonts.googleapis.com 为DIRECT 中 会导致访问部分网站显示异常。

**Comment by kkkgo** [comment]
你把fast_node功能开起来就会忽略规则了。

**Reply by ghost** [reply]
但是又有 APP 走不同节点的需求。
网页 IEPL 
动画疯 TW
NF HK
所以 希望 有选项 剔除 yaml 或者suburl 中的 DIRECT 规则

**Comment by Lovest20018** [comment]
不要 用机场的规则-自己手搓下好了-

**Comment by kkkgo** [comment]
ppgw不能为你决定使用或者不使用某条规则，建议手动写适合自己的规则，或者使用subconverter等程序转换定制订阅。
`fonts.googleapis.com`国内的确有镜像，只是连通性不一定好使，如果你使用PaoPaoDNS可以尝试把`fonts.googleapis.com`加入force_cn_list。

**Comment by ghost** [comment]
我自己糊一下吧
